sdk/managedservices/Microsoft.Azure.Management.ManagedServices/AzSdk.RP.props

```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
   <PropertyGroup>
-    <AzureApiTag>ManagedServices_2018-06-01-preview;</AzureApiTag>
+    <AzureApiTag>ManagedServices_2019-06-01;</AzureApiTag>
```

Comment:
Please update the value of **Version** in Microsoft.Azure.Management.ManagedServices.csproj